### PR TITLE
Feature branch popup window

### PR DESCRIPTION
**Pull request by bertinkambale**
- [ ]  I implemented the following interactions:
- [ ] When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- [ ]  When the user clicks (or taps) the close (X) button, the popup disappears.
- [ ] In order to associate each project with the popup window details, you must refactor the project section:
- [ ] You need to use **a JavaScript array to store all of the information for all projects**.
- [ ] For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
- [ ] You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
- [ ] After that, you could implement the popup window.
- [ ] You should implement popups for both mobile and desktop screen sizes:
- [ ] Mobile popup

### CODE REVIEW

**Review by ha-manel — COMMENTED** [review]
# STATUS: INVALID 🔴 
Hi 
I'm sure you put a lot of effort into this project but this PR is invalid because Eslint wasn't setup😓 

For this project and any project that uses JavaScript: Eslint, Webhint, Lighthouse, and Stylelint have to be set up correctly 👍 
Kindly update your `linters.yml` file and add any necessary files then submit the PR for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                </ul>
+                <img src="${projects.img2}">
+                <div class="popup-content">
+                    <div class="left-section">
+                        <p class="project-descritpion">${projects.description2}</p>
+                    </div>
+                    <div class="right-section">
+                        <ul class="stacks popup">
+                            <li class="stack">Html</li>
+                            <li class="stack">Css</li>
```
Comment (sja-thedude) [inline script.js:123]: - [ ] Great job so far, Kindly implement small changes in CSS as per the attached loom video.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @bertinkambale, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters passing
✔️ Correct gitflow
✔️ Professional Readme

## Required 
- [ ] Kindly make sure to implement the changes as per the attached loom video [here](https://www.loom.com/share/85534e2eda4440fa935094161f458e3e).

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi guys, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
almost there

 ## Highlights
-All linter checks are passing correctly ✔️ 
- Descriptive PR summary and title 👍 

 ## Required Changes ♻️
- Just like the previous reviewer suggested in the video, some parts of your popup window aren't precisely like it is displayed in the Figma template. Remember it is essential that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project(The header and the tech stack are not properly aligned, please fix this):
popup header


_Check the comments under the review._

## Optional suggestions
**OPTIONAL** - There seems to be an issue with the icons in your project. You seem to be navigating into the wrong path, hence your icons aren't displayed properly. Your icon folder is named `Icons` and I see you are using `ICONS`. Please fix this 👍 👍 

Your project:
buttons


Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your code.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**